### ISSUE TITLE
Redmine 6.0.2 - NameError: undefined local variable or method `unloadable' for class DocumentsController (NameError)

### ISSUE THREAD

**Issue by smoreau** [docker-library/redmine#355]
I am getting the following error when running Redmine 6.0.2 via docker compose using the Docker Official image: https://hub.docker.com/_/redmine 

What should I do?

```
redmine  | Your Gemfile lists the gem puma (>= 0) more than once.
redmine  | You should probably keep only one of them.
redmine  | Remove any duplicate entries and specify the gem only once.
redmine  | While it's not a problem now, it could cause errors if you change the version of one of them later.
redmine  | The Gemfile's dependencies are satisfied
redmine  | rake aborted!
redmine  | NameError: undefined local variable or method `unloadable' for class DocumentsController (NameError)
redmine  | 
redmine  |       unloadable # Send unloadable so it will not be unloaded in development
redmine  |       ^^^^^^^^^^
redmine  | /usr/src/redmine/plugins/redmine_document_move/lib/documents_controller_patch.rb:14:in `block in included'
redmine  | /usr/src/redmine/plugins/redmine_document_move/lib/documents_controller_patch.rb:13:in `class_eval'
redmine  | /usr/src/redmine/plugins/redmine_document_move/lib/documents_controller_patch.rb:13:in `included'
redmine  | /usr/src/redmine/plugins/redmine_document_move/lib/documents_controller_patch.rb:32:in `include'
redmine  | /usr/src/redmine/plugins/redmine_document_move/lib/documents_controller_patch.rb:32:in `<top (required)>'
redmine  | /usr/local/bundle/gems/zeitwerk-2.7.1/lib/zeitwerk/core_ext/kernel.rb:34:in `require'
redmine  | /usr/local/bundle/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
redmine  | /usr/src/redmine/plugins/redmine_document_move/init.rb:4:in `<top (required)>'
redmine  | /usr/src/redmine/lib/redmine/plugin_loader.rb:31:in `load'
redmine  | /usr/src/redmine/lib/redmine/plugin_loader.rb:31:in `run_initializer'
redmine  | /usr/src/redmine/lib/redmine/plugin_loader.rb:61:in `each'
redmine  | /usr/src/redmine/lib/redmine/plugin_loader.rb:61:in `block in load'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:407:in `instance_exec'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:407:in `block in make_lambda'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:179:in `block in call'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:668:in `block (2 levels) in default_terminator'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:667:in `catch'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:667:in `block in default_terminator'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:180:in `call'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:559:in `block in invoke_before'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:559:in `each'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:559:in `invoke_before'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/callbacks.rb:109:in `run_callbacks'
redmine  | /usr/local/bundle/gems/activesupport-7.2.2/lib/active_support/reloader.rb:96:in `prepare!'
redmine  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/application/finisher.rb:74:in `block in <module:Finisher>'
redmine  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/initializable.rb:32:in `instance_exec'
redmine  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/initializable.rb:32:in `run'
redmine  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/initializable.rb:61:in `block in run_initializers'
redmine  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/initializable.rb:60:in `run_initializers'
redmine  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/application.rb:435:in `initialize!'
redmine  | /usr/src/redmine/config/environment.rb:16:in `<top (required)>'
redmine  | /usr/local/bundle/gems/zeitwerk-2.7.1/lib/zeitwerk/core_ext/kernel.rb:34:in `require'
redmine  | /usr/local/bundle/gems/polyglot-0.3.5/lib/polyglot.rb:65:in `require'
redmine  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/application.rb:411:in `require_environment!'
redmine  | /usr/local/bundle/gems/railties-7.2.2/lib/rails/application.rb:559:in `block in run_tasks_blocks'
redmine  | /usr/local/bundle/gems/rake-13.2.1/exe/rake:27:in `<top (required)>'
redmine  | Tasks: TOP => db:migrate => db:load_config => environment
redmine  | (See full trace by running task with --trace)
redmine exited with code 0
```

Here is my docker-compose.yml file:
```
  redmine:
    image: redmine:6.0.2-alpine
    container_name: redmine
    depends_on:
      - mariadb
    ports:
      - 8080:3000
    environment:
      - REDMINE_DB_MYSQL=mariadb
      - REDMINE_DB_USERNAME=redmine_user
      - REDMINE_DB_PASSWORD=redmine_pwd
      - REDMINE_DB_DATABASE=logikdev_redmine
    restart: always
```

**Comment by ishikawa999** [comment]
@smoreau 
The `unloadable` notation has not been available since Redmine 3.1.0( https://www.redmine.org/issues/20513 ), regardless of this Docker image. It seems you are either using an old version of the plugin or the plugin is not compatible with Redmine 6. Please try commenting out the line with unloadable and see if that resolves the issue.

**Comment by smoreau** [comment]
You are absolutely right, I didn't know this notation has not been available since Redmine 3.1.0. The plugin was working until Redmine 5 though.
Anyway, I fixed the faulty plugin and Redmine is starting properly now. :)
Thank you for your help on this issue.
